File: the-big-fan/lambdas/subscribe/createdStatus.ts
export {};

exports.handler = async function(event:any) {
  console.log("request:", JSON.stringify(event, undefined, 2));

  let records: any[] = event.Records;
  
  for(let index in records) {
    let payload = records[index].body;
    console.log('received message ' + payload);
  }
};

File: the-big-fan/python/lambda_fns/subscribe/createdStatus.py
import json
import logging

def handler(event, context):
    logger = logging.getLogger(__name__)
    logger.setLevel(logging.INFO)
    logger.info("request: " + json.dumps(event))

    records = event["Records"]
    
    for record in records:
        payload = record["body"]
        logger.info("received message " + payload)